src/t2iapi/activation_state/activation_state_requests.proto

```diff
@@ -34,6 +34,19 @@ message SetAlertActivationRequest {
     AlertActivation activation = 2;  // value to set as the ActivationState value
 }
 
+/*
+Request to set the ActivationState of the component or metric with the given handle and the start and
```

Comment:
misleading description, the objective of the manipulation is not very clear, write the expectancy
transition from one state to another is not mentioned